gui/src/advancedDynamicTexture.ts

```diff
@@ -88,6 +89,20 @@ module BABYLON.GUI {
             this._rootContainer._markAllAsDirty();
         }
 
+        public get useSmallestIdeal(): boolean {
+            return this._useSmallestIdeal;
+        }
+
+        public set useSmallestIdeal(value: boolean) {
+            if (this._useSmallestIdeal === value) {
+                return;
+            }
+
+            this._useSmallestIdeal = value;

```

Comment:
Do you mind using line returns and brackets to make it easier to read

Reply:
Feels a bit cluttered to me but I guess something like that?